src/mmap_unix.rs

```diff
@@ -210,15 +217,16 @@ impl MmapRegion {
 }
 
 impl AsSlice for MmapRegion {
-    // Returns the region as a slice
-    // used to do crap
+    /// Returns the region as a slice.
```

Comment:
This should also have a `#Safety` section.

Reply:
Fixed.